api/src/main/resources/openmrs-distro.properties

```diff
@@ -26,11 +29,10 @@ omod.reporting=${reportingVersion}
 omod.reportingcompatibility=${reportingcompatibilityVersion}
 omod.reportingrest=${reportingrestVersion}
 omod.reportingui=${reportinguiVersion}
-omod.serialization.xstream=${serializationVersion}
 omod.serialization.xstream.type=omod
-omod.sync=${syncVersion}
```

Comment:
Why remove the sync module from the distribution?

Reply:
I added it back.